app/models/foreman_openscap/oval_status.rb

```diff
@@ -0,0 +1,38 @@
+module ForemanOpenscap
+  class OvalStatus < ::HostStatus::Status
+    PATCHED = 0
+    VULNERABLE = 1
```

Comment:
Ideally we'd have 3 different states

OK - no CVEs found (0 CVEs records for the host)
Warning - CVEs found, no errata available for any of them
Error - CVEs found, errata for them exist (patch available, but not applied)